src/inference/src/dev/threading/cpu_streams_executor_internal.cpp

```diff
@@ -25,6 +25,7 @@ void get_cur_stream_info(const int stream_id,
     int stream_total = 0;
     size_t stream_info_id = 0;
     bool cpu_reserve = cpu_reservation;
+    bool pcore_only = true;
```

Comment:
Please change the parameter name to `ecore_used` for following logic. 

current `pcore_only = true` will confuse developer on Ecore only case.

Reply:
Ok, updated.